src/sql/src/func.rs

```diff
@@ -1177,6 +1177,18 @@ pub enum Func {
     Table(Vec<FuncImpl<TableFuncPlan>>),
 }
 
+/// Functions using this macro should be transformed/planned away before
+/// reaching function selection code, but still need to be present in the
+/// catalog during planning.∑
```

Comment:
stray sigma at the end of the line here :D

Reply:
Can't trust Irma on your lap while you're working